pkg/gktest/filter.go

```diff
@@ -39,21 +39,21 @@ func NewFilter(run string) (Filter, error) {
 	return Filter{}, nil
 }
 
-// MatchesTest filters the set of constraint tests to run by constraint name
+// MatchesConstraint filters the set of constraint tests to run by constraint name
 // and the tests contained in the constraint. Returns true if tests in the constraint
 // should be run.
 //
 // If a constraint regex was specified, returns true if the constraint regex
 // matches `constraint`.
 // If a constraint regex was not specified but a test regex was, returns true if
 // at least one test in `tests` matches the test regex.
-func (f Filter) MatchesTest(c Test) bool {
+func (f Filter) MatchesConstraint(c Test) bool {
```

Comment:
This is re-introducing the "MatchesConstraint" name.

Reply:
Fixed.